templates/steering-display/src/tft_data.h

```diff
@@ -0,0 +1,11 @@
+
+#ifndef TFT_DATA_H
+#define TFT_DATA_H
+
+#if !defined(PROGMEM)
+	#define PROGMEM
+#endif
+
+extern const uint8_t logo[5298] PROGMEM;
```

Comment:
is having full CPP useful to us? I think with the .c extension PIO will compile this as straight C. if CPP features are useful you can change the file extensions and change this extern to an extern C to prevent name mangling.